modules/documents/client/directives/documents.manager.link.directive.js

```diff
@@ -229,12 +229,12 @@ angular.module('documents')
               $scope.project = scope.project;
               $scope.authentication = Authentication;
 
-              self.title = "Link Documents to '" + $scope.project.name + "'";
+              self.title = "Add Documents to '" + $scope.project.name + "'";
               if (!_.isEmpty(scope.collection.displayName)) {
-                self.title = "Link Documents to '" + scope.collection.displayName + "'";
+                self.title = "Add Documents to '" + scope.collection.displayName + "'";
               }
 
-              self.linkedFiles = angular.copy(scope.target || []);
```

Comment:
If target is not being used anymore, it should be removed from the directive parameters as well [line 211](https://github.com/bcgov/mem-admin/pull/238/commits/4d25f2b7ce1bf4ad0514a9b4b08ba9c1a5913019#diff-85454c483c32fbf262dd381a6e5c311cL211)